<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="0"
            tests="0">

  <testsuite name="badDatatest"
    errors="0" failures="0" hostname="" id="" package="badDatatest" skipped="0"
    tests="0" time="0.000" timestamp="Mon, 06 Nov 2017 16:57:20 GMT">
  

  
    <system-err>
        Error while running [About Us / test]:

  TypeError: Cannot read property 'back' of undefined
    at Object.test (C:\Users\tyler\Desktop\DevMt\tyler-el-sistema-automation\tests\aboutUs.js:23:19)
    at Module.call (C:\Users\tyler\AppData\Roaming\npm\node_modules\nightwatch\lib\runner\module.js:62:34)
    at C:\Users\tyler\AppData\Roaming\npm\node_modules\nightwatch\lib\runner\testcase.js:70:29
    at _fulfilled (C:\Users\tyler\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:834:54)
    at self.promiseDispatch.done (C:\Users\tyler\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:863:30)
    at Promise.promise.promiseDispatch (C:\Users\tyler\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:796:13)
    at C:\Users\tyler\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:556:49
    at runSingle (C:\Users\tyler\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:137:13)
    at flush (C:\Users\tyler\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:73:7)
    </system-err>
  

  
  </testsuite>
</testsuites>
